lib/experimental/commands.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Bootstraps the Command Palette  in the admin dashboard.
+ *
+ * @package Gutenberg
+ */
+
+/**
+ * Enqueues the assets required for the Command Palette.
+ *
+ * @global string $pagenow The name of the current admin page being viewed.
+ */
+function gutenberg_enqueue_command_palette_assets() {
+	// For now, we don't enqueue assets to the frontend. It may be available in the future.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	global $pagenow, $current_screen;
+
+	// The Site Editor and Post Editor already implement the Command Palette
+	// within the app, so do nothing on those pages.
+	if ( in_array( $pagenow, array( 'site-editor.php', 'post.php', 'post-new.php' ), true ) && $current_screen->is_block_editor() ) {
```

Comment:
Should we remove it from the editors and avoid this check?

Reply:
Do you think we can follow-up with this?

Reply:
My explanation wasn't clear enough.

If we remove this check, we'll need to remove the command in the `edit-post` and `edit-site` to prevent duplicate registration of the command.

However, removing the command from these two packages will mean that the command won't work when the two packages are bundled into core.

However, if we can decide to ship this PR into WP 6.9, we can remove this check along with the core backport PR.

I hope I explained it well 😅

Reply:
Nothing is stopping us from doing multiple package updates no?

Reply:
I see, I think I might be misunderstanding 😅 I'd like to submit a follow-up PR to remove this check soon.